include/fuzzuf/optimizer/pso.hpp

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Is this file still needed?

Reply:
This is not required for MOpt, just for reference implementation of original PSO as some parts of MOpt copied from here.

Reply:
`class Particle` is used in MOptOptimizer which is implemented in PSO.